Projects/My_website/myWebsite/blog/views.py

```diff
@@ -119,7 +119,7 @@ def post(self, request):
             blog.save()
             return Response({"you liked this blog"})
         except:
-            return Response({"message": "Provided blog-id is not valid"})
+            raise Exception("provided blog-id is not valid")
```

Comment:
@codal-uchoudhary 
It should be something like this

except Exception as e:
  raise e